packages/application-shell/src/components/public-page-container/public-page-container.js

```diff
@@ -1,31 +1,96 @@
 import PortalsContainer from '../portals-container';
-import styles from './public-page-container.mod.css';
 import messages from './messages';
 
 const year = new Date().getUTCFullYear();
 
-const PublicPageContainer = props => (
-  <React.Fragment>
-    <PortalsContainer />
-    <div className={styles.container}>
-      <Spacings.Stack scale="m">
-        {props.children}
-        <Spacings.Inline justifyContent="space-between">
-          <a href={`https://commercetools.com/privacy`} target="_blank">
-            <Text.Detail>
-              <FormattedMessage {...messages.privacyPolicy} />
-            </Text.Detail>
-          </a>
-          <Text.Detail>{`${year} © commercetools`}</Text.Detail>
-        </Spacings.Inline>
-      </Spacings.Stack>
-    </div>
-  </React.Fragment>
-);
+const PublicPageContainer = props => {
+  const privacyPolicyLink = (
+    <a href={`https://commercetools.com/privacy`} target="_blank">
+      <FormattedMessage {...messages.privacyPolicyLink} />
+    </a>
+  );
+
+  const termsOfServiceLink = (
+    <a href={`https://commercetools.com/terms`} target="_blank">
+      <FormattedMessage {...messages.termsOfServiceLink} />
+    </a>
+  );
+
+  return (
+    <React.Fragment>
+      <PortalsContainer />
+      <div
+        css={css`
+          height: 100vh;
+          display: flex;
+          padding-top: ${customProperties.spacing32};
+          justify-content: center;
+          background-color: #073538;
+          background-image: url('${BackgroundImage}');
+        `}
+      >
+        <Spacings.Stack scale="xl" alignItems="center">
+          <Spacings.Stack scale="m" alignItems="center">
+            <img
+              css={css`
+                width: 275px;
+                height: 45px;
+              `}
+              src={LogoBWWhiteSVG}
+            />
+            <div>
+              <Text.Headline elementType="h2">
+                <span style={{ color: 'white' }}>
+                  <FormattedMessage {...messages.welcome} />
+                </span>
+              </Text.Headline>
+            </div>
+          </Spacings.Stack>
+          <Spacings.Stack scale="s">
+            {/* Login Card */}
+            {props.children}
+            <div
+              css={css`
+                display: flex;
+                align-items: center;
+                justify-content: space-between;
+              `}
+            >
+              <Spacings.Stack scale="xs">
+                <div>
+                  <Text.Detail>
+                    <span style={{ color: 'white' }}>
+                      <FormattedMessage
+                        {...messages.termsAndPrivacy}
+                        values={{ privacyPolicyLink, termsOfServiceLink }}
+                      />
+                    </span>
+                  </Text.Detail>
+                </div>
+                <Text.Detail>
+                  <span style={{ color: 'white' }}>
```

Comment:
can you use css in js here instead of inline style?

Reply:
It's feels to me like this is a hack around the fact that our Text components can't do white colours. Should they maybe?

Reply:
Yes, I added these before adding `emotion` and they're only here as a "hack" until we have proper theming.
Maybe I should just use css in js and add a comment to remind to remove the hack later?

Reply:
There is an `inverted` tone for the white color

Reply:
can't you just do at the top of your file (outside the react component)
```
const white = css`color: white`
```

And then do css={white} on the spans?

Reply:
yeah you can use `<Text.Body tone="inverted" >`

Reply:
Ah cool, I wasn't aware!

Reply:
Is there any reason why only the `Text.Body` component has the "inverted" tone? (and not also the Headline, Subheadline and Detail)

Reply:
🤷‍♂️ . If there's a use case for it, we can add it. I doubt there is though.

Reply:
Just the Headline of this page. Right now it's solved with the styled span wrapper

Reply:
If it's just one in one place in the whole MC, I would stick with your approach 👍